### PR TITLE
Added LICENSE file to repo and added LICENSE info to README.md

### LINKED ISSUE #17: Add LICENSE file, and add LICENSE header to all code files.

### PR DESCRIPTION
**Pull request by evanSpendlove**
Closes #17.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
 # Hermes 
+
+**This is not an officially supported Google product.**
+
 Hermes is an awesome project!
```
Comment (deniadelcid) [inline README.md]: Although this is 100% true, lets replace this with a TODO to add more information on how to bring up hermes and how to get started with it.

**Review by deniadelcid — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (evanSpendlove) [inline README.md]:
```suggestion
TODO: Add more information on Hermes setup, running Hermes, etc.
```

Comment (evanSpendlove) [reply README.md]: I've addressed this with a suggestion and new commit.

**Review by alicjakwie — APPROVED** [review]
just a comment that we need to remember to paste the Apache License Header into all of our code files - maybe create an issue to verify that

-- Reply --
I've created an issue for this :)

**Pull request merged**